eng/common/scripts/Delete-RemoteBranches.ps1

```diff
@@ -44,16 +66,21 @@ foreach ($res in $responses)
     LogError "Get-GitHubPullRequests failed with exception:`n$_"
     exit 1
   }
-
   $openPullRequests = $pullRequests | ? { $_.State -eq "open" }
-  if ($openPullRequests.Count -gt 0)
```

Comment:
If we don't have a CentralRepoId then we should still do this check.

Reply:
Perhaps I'm missing something but it doesn't look like you are handling this case any longer. If we don't have a centralrepoid and we find a matching branch but that branch has an open PR against it we shouldn't delete it.

Reply:
I moved the piece of check in the another helper function.

Reply:
Put the function inline.